src/InfrastructureLayer/Settings/Settings.cpp

```diff
@@ -26,7 +26,32 @@ Settings::Settings(QString filepath)
 {
     if (settings_.value(IP_ADDRESS).isNull())
     {
```

Comment:
I think you missed `QUEUE_NAME `

Reply:
As you mentioned, the queue name is set in `EpsilonDashboard.cpp` when the software starts up.